src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -423,7 +437,14 @@ function countDescendantFrames(batch: RenderBatch, frame: RenderTreeFrame): numb
   }
 }
 
-function raiseEvent(event: Event, browserRendererId: number, eventHandlerId: number, eventArgs: EventForDotNet<UIEventArgs>, eventFieldInfo: EventFieldInfo | null) {
+function raiseEvent(
+  eventDispatcher: ((descriptor: EventDescriptor, args: UIEventArgs) => Promise<void>),
```

Comment:
Hopefully this extra arg can be removed if we're not passing around `eventDispatcher` instances as per other comments.

Reply:
I got rid of this in favor of a global function